docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Getting started with Grafana Mimir using the Helm chart"
+menuTitle: "Getting started using the Helm chart"
+description: "Learn how to get started with Grafana Mimir using the Helm chart."
+weight: 20
+---
+
+# Getting started with Grafana Mimir using the Helm chart
+
+<!-- What is this about? TODO -->
+
+## Before you begin
+
+The instructions that follow are common across any flavor of Kubernetes.
+
+The information that follows assumes that you are able to install Kubernetes, and configure and operate it.
+
+Install the following software:
+
+- Either a [Prometheus server](https://prometheus.io/docs/prometheus/latest/installation/) or [Grafana Agent](https://grafana.com/docs/grafana-cloud/agent/#installing-the-grafana-agent).
+<!-- TBD, Krajo to figure out: - Verify that you have enough memery overall and number of cores. Krajo needs a couple of days for this answer to come in. -->
```

Comment:
A minimal choice is to say that you have 2 cores (2000m) and 8GiB memory available, which means a laptop with something like 4 cores and 16 GiB RAM is enough to get started, but this is not recommended for production as in production we start from 3 nodes(!) for redundancy.